src/routes/(app)/settings/SubscriptionTags.svelte

```diff
@@ -12,7 +13,11 @@
     .filter((t) => t != "");
 </script>
 
-<div class="flex flex-col items-center sm:grid sm:grid-cols-2">
+<div
+  class={languageTag() == "sv"
+    ? "flex flex-col items-center md:grid md:grid-cols-2"
+    : "flex flex-col items-center xl:grid xl:grid-cols-2"}
```

Comment:
Is this really the best way to do this? Feels kinda sketchy

Reply:
It's pretty sketchy, the english names are usually longer than the swedish ones but it either looks pretty empty or cramped if you only have 1 of them. Could otherwise decide on a longer settings page by not having notifications beside tags, but there will be a bit more scrolling if that's ok.

Reply:
Fair enough. It looks fine when I look at it, so it's not outright wrong.

Reply:
I think we should find some solution with auto columns, min-content or something like that. We never know what the tags will have as names in the future.